src/components/Publish/Pricing/Price.tsx

```diff
@@ -59,12 +58,17 @@ export default function Price({
                 <Conversion price={field.value} className={styles.conversion} />
               </h4>
             </div>
-
-            <h4 className={styles.datatoken}>
-              = <strong>1</strong> {dataTokenOptions.symbol}{' '}
-              <Conversion price={field.value} className={styles.conversion} />
-            </h4>
           </div>
+          {firstPrice && (
```

Comment:
there is no `firstPrice` anymore so this can be removed again, same for respective CSS rules.

Reply:
cool ,didn't know what was that for